iceoryx_utils/include/iceoryx_utils/internal/posix_wrapper/access_control.hpp

```diff
@@ -56,11 +56,11 @@ class AccessController
     };
 
     /// @brief access right for a permission entry
-    enum class Permission
+    enum class Permission : acl_perm_t
     {
         READ = ACL_READ,
         WRITE = ACL_WRITE,
```

Comment:
@rotu Then we should also replace READ and WRITE with Permission::READ and Permission::WRITE ?

Reply:
I don't totally understand what you mean.

In `READWRITE = Permission::READ | Permission::WRITE`, the `Permission::` is redundant, but I added it to visually distinguish where we're pulling in platform header constants `ACL_READ` from where we're defining `READWRITE` as a composite of other bit flags.

Reply:
Sorry, now I got it. I thought you introduce something else from acl.h. But it is the own enum class (and acl would be C anyway)